timm/models/efficientvit_mit.py

```diff
@@ -596,6 +776,129 @@ def forward(self, x):
         return x
 
 
+class EfficientVitLarge(nn.Module):
+    def __init__(
+        self,
+        in_chans=3,
+        widths=(),
+        depths=(),
+        head_dim=32,
+        norm_layer=nn.BatchNorm2d,
+        act_layer=nn.GELU,
+        global_pool='avg',
+        head_widths=(),
+        drop_rate=0.0,
+        num_classes=1000,
+        eps=1e-7,
+    ):
+        super(EfficientVitLarge, self).__init__()
+        self.grad_checkpointing = False
+        self.global_pool = global_pool
+        self.num_classes = num_classes
+
+        # input stem
+        self.stem = Stem(in_chans, widths[0], depths[0], norm_layer, act_layer, block_type='large')
+        stride = self.stem.stride
+
+        # stages
+        self.feature_info = []
+        self.stages = nn.Sequential()
+        in_channels = widths[0]
+        for i, (w, d) in enumerate(zip(widths[1:], depths[1:])):
+            self.stages.append(EfficientVitLargeStage(
+                in_channels,
+                w,
+                depth=d,
+                norm_layer=norm_layer,
+                act_layer=act_layer,
+                head_dim=head_dim,
+                vit_stage=i >= 3,
+                fewer_norm=i >= 2,
+            ))
+            stride *= 2
+            in_channels = w
+            self.feature_info += [dict(num_chs=in_channels, reduction=stride, module=f'stages.{i}')]
+
+        self.num_features = in_channels
+        self.head_widths = head_widths
+        self.head_dropout = drop_rate
+        if num_classes > 0:
+            self.head = ClassifierHead(
+                self.num_features,
+                self.head_widths,
+                n_classes=num_classes,
+                dropout=self.head_dropout,
+                global_pool=self.global_pool,
+                act_layer=act_layer,
+            )
+        else:
+            if self.global_pool == 'avg':
+                self.head = SelectAdaptivePool2d(pool_type=global_pool, flatten=True)
+            else:
+                self.head = nn.Identity()
+        self.set_norm_eps(eps)
+
+    @torch.jit.ignore
+    def set_norm_eps(self, eps):
+        for m in self.modules():
```

Comment:
1e-7 is a weird norm eps, odd choice. 1e-6, 1e-5, or 1e-3 covers 99% of the models I've ever seen.

Anyways, for most other models this is handled by having a norm_layer arg as already exists, and then binding norm eps to norm_layer via partial before passing through the rest of the modules. Might be better to follow that than use this fn, as it will not work with any custom norm layer passed that isn't in the isinstance list